lib/src/validationsDispatcher.ts

```diff
@@ -59,20 +61,21 @@ export class ValidationDispatcher {
   }
 
   fireAllFieldsValidations(
-    vm: Object,
+    vm: any,
+    vmKeys: string[],
```

Comment:
That's because your `vm` could have more keys/value that keys which needs to be validated, for instance, suppose we have an entity of 12 variables which only 5 needs validation. We need an array of fields with validations to iterate (and that is stored by ValidationEngine).